OWML.Patcher/GameVersionReader.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OWML.Common;
+using System;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Patcher
+{
+    public class GameVersionReader
+    {
+        private readonly IModConsole _writer;
+        private readonly BinaryPatcher _binaryPatcher;
+
+        private const int PlayerSettingsSector = 0;
+
+        public GameVersionReader(IModConsole writer, BinaryPatcher binaryPatcher)
+        {
+            _writer = writer;
+            _binaryPatcher = binaryPatcher;
+        }
+
+        public string GetGameVersion()
+        {
+            var sectorBytes = _binaryPatcher.GetSectorBytes(_binaryPatcher.ReadFileBytes(), PlayerSettingsSector);
+            for (int i = 4; i < sectorBytes.Length - 2; i++)
+            {
+                if (sectorBytes[i - 4] == 0 && sectorBytes[i - 3] == 0 && sectorBytes[i - 2] == 0 &&
+                    sectorBytes[i] == '.' && sectorBytes[i + 2] == '.')
```

Comment:
Are we looking for [0, 0, 0, ?, '.', ?, '.']? Why, what's special about it? Seems fragile...

Reply:
I don't think first and second numbers would become two-digit anytime soon, as well as version string getting longet than 255 chars :P

Reply:
or should I just do "two dots with character in-between" check?

Reply:
Dunno but this is hard to read.
Can the whole thing be converted into a string, and then you look for the substring instead? That would be so much easier.

Reply:
simple substring won't work

Reply:
regex is nice though